apps/server/src/modules/deletion/deletion-api.module.ts

```diff
@@ -0,0 +1,45 @@
+import { Module } from '@nestjs/common';
+import { DeletionModule } from '@modules/deletion';
+import { AccountModule } from '@modules/account';
+import { ClassModule } from '@modules/class';
+import { LearnroomModule } from '@modules/learnroom';
+import { FilesModule } from '@modules/files';
+import { PseudonymModule } from '@modules/pseudonym';
+import { LessonModule } from '@modules/lesson';
+import { TeamsModule } from '@modules/teams';
+import { UserModule } from '@modules/user';
+import { LoggerModule } from '@src/core/logger';
+import { AuthenticationModule } from '@modules/authentication';
+import { RocketChatUserModule } from '@modules/rocketchat-user';
+import { Configuration } from '@hpi-schul-cloud/commons';
+import { RocketChatModule } from '@modules/rocketchat';
+import { DeletionRequestsController } from './controller/deletion-requests.controller';
+import { DeletionExecutionsController } from './controller/deletion-executions.controller';
+import { DeletionRequestUc } from './uc';
+
+@Module({
+	imports: [
+		DeletionModule,
```

Comment:
When you import the deletionModule than all the other modules should be possible to remove. They should part of the deleltion module and it should not exists a need to use it in api module directly or?

@CeEv check usage of it

Reply:
Fine for now. But must be switch by rewrite.

Reply:
so done for now